common/source/docs/common-yjuav-a6se.rst

```diff
@@ -0,0 +1,291 @@
+3                 CAN_N                +3.3V
+4                 GND                  GND
+=============     ================     =============
+
+GPS1
+----------
+=============     ================     =============
+Pin               Signal               Volt
+=============     ================     =============
+1                 VCC                  +5V
+2                 UART_TX              +3.3V
+3                 UART_RX              +3.3V
+4                 I2C_SCL              +3.3V
+5                 I2C_SDA              +3.3V
+6                 GND                  GND
+=============     ================     =============
+
+GPS2&SAFETY
+----------
+=============     ================     =============
+Pin               Signal               Volt
+=============     ================     =============
+1                 VCC                  +5V
+2                 UART_TX              +3.3V
+3                 UART_RX              +3.3V
+4                 I2C_SCL              +3.3V
+5                 I2C_SDA              +3.3V
+6                 SAFETY_SW            +3.3V
+7                 SAFETY_SW_LED        +3.3V
+8                 3V3_OUT              +3.3V
+9                 BUZZER               +3.3V
+10                GND                  GND
+=============     ================     =============
+
+DEBUG
+----------
+=============     ================     =============
+Pin               Signal               Volt
+=============     ================     =============
+1                 VCC                  +5V
+2                 TX                   +3.3V
+3                 RX                   +3.3V
+4                 SWDIO                +3.3V
+5                 SWCLK                +3.3V
+6                 GND                  GND
+=============     ================     =============
+
+SAFETY
+----------
+=============     ================     =============
+Pin               Signal               Volt
+=============     ================     =============
+1                 3V3_OUT              +3.3V
+2                 SAFETY_SW            +3.3V
+3                 SAFETY_SW_LED        +3.3V
+4                 SBUS_OUT             +3.3V
+5                 RSSI                 +3.3V
+6                 GND                  GND
+=============     ================     =============
+
+USB EX
+--------
+=============     ================     =============
+Pin               Signal               Volt
+=============     ================     =============
+1                 VCC_IN               +5V
+2                 DM                   +3.3V
+3                 DP                   +3.3V
+4                 GND                  GND
+=============     ================     =============
+
+UART Mapping
+============
+
+- SERIAL0 -> USB (OTG1)
+- SERIAL1 -> USART1 (Telem1)
+- SERIAL2 -> USART6 (Telem2)
+- SERIAL3 -> USART3 (GPS1), NODMA
+- SERIAL4 -> USART2 (GPS2), NODMA
+- SERIAL5 -> UART8 (SBUS)
```

Comment:
where is UART8 pinned out? its say SBUS....is the SBUS out? TX only?

Reply:
yes, This is defined as SBUS.out and only TX.

Reply:
where in the hwdef or default params do you actually set the UART8 protocol to SBUS servos out?have you tested this???

Reply:
Yes, I have tested this output and the data is normal.
100Kps, 8E2
![image](https://github.com/ArduPilot/ardupilot_wiki/assets/116018265/fdd6de4b-4900-4f56-98b8-d86fd01b53ce)

Reply:
> 

I didn't explain this serial port well above. In fact, the UART8 RX is also connected to the RC IN, and the SBUS OUT is connected to the SAFE identification interface. You can see it there, so the explanation will be clearer. So UART8 is input&output.

Reply:
I see SBUS_OUT on the SAFETY connector and understand that its UART8TX, but SERIAL5's default protocol is NONE, so it will not output SBUS on its TX, nor will it detect RC on its RX input, even if tied to the RCIN pin...so a user will hook up an SBUS device to the SBUS_OUT and it will do nothing, unless he sets SERIAL5 protocol....this will result in customer support calls...that is why I suggest that a defaults.param file be added to the code to set it by default to enable SBUS outputs on UART8TX....in the wiki page we can explain an alternate use as RCIN (with a protocol change and losing the SBUS out feature)....since the UART8 is tied to the RCIN pin, its use as a general purpose UART is prevented and that also needs explanation in the wiki page....

Reply:
How can you help modify the wiki content? The default parameters will not be modified. The wiki explains how this UART is used by users.

Reply:
I will work on the page today....also need to know which I2C bus is on the I2C connector and which is on the GPS connector? (1/2/3 or 4)

Reply:
just fyi, your readme and this page have errors regarding SERIAL port vs UART...which I am correctly in this page also